app/scripts/xprizeDemo.xml

```diff
@@ -0,0 +1,243 @@
+    <dependencies>
+      <port timeout="5.0" request="start" reply="[ok]">/icub-virtualizer/microphone/rpc</port>
+    </dependencies>
+  </module>
+
+  <!-- icub-console -->
+
+  <module>
+    <name>camCalibWithPose</name>
+    <parameters>--context cameraCalibration --from icubEyes.ini --group CAMERA_CALIBRATION_LEFT --name /icub/camcalib/left</parameters>
+    <node>icub-console</node>
+  </module>
+
+  <module>
+    <name>camCalibWithPose</name>
+    <parameters>--context cameraCalibration --from icubEyes.ini --group CAMERA_CALIBRATION_RIGHT --name /icub/camcalib/right</parameters>
+    <node>icub-console</node>
+  </module>
+
+  <!-- icub-virtualizer -->
+  <module>
+    <name>yarpdev</name>
+    <parameters> --device JoypadControlServer --use_separate_ports 1 --period 10 --name /joypadDevice/Oculus --subdevice ovrheadset --sticks 0  --tfDevice transformClient --tfLocal /oculustf --tfRemote /transformServer --tf_head_frame headoculus --tf_left_hand_frame loculus --tf_right_hand_frame roculus --tf_root_frame oculusworld --stick_as_axis true --userpose true --gui_elements 1 --GUI_0::width 0.15 --GUI_0::height 0.15 --GUI_0::x 0.2 --GUI_0::y -0.2 --GUI_0::z -0.5 --GUI_0::alpha 1.0 --hfov 59 --left-pitch-offset 0.0725 --left-yaw-offset -0.065 --right-pitch-offset 0.13 --right-yaw-offset 0.07</parameters>
+    <node>icub-virtualizer</node>
+  </module>
+
+  <module>
+    <name>VirtualizerModule</name>
+    <node>icub-virtualizer</node>
+    <parameters> --context virtualizerRetargeting </parameters>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters> --device AudioRecorderWrapper --subdevice portaudioRecorder --name /icub-virtualizer/microphone  --min_samples_over_network 4000  --max-samples_over_network 4000  --rate 160000 --samples 16000 --channels 1</parameters>
+    <node>icub-virtualizer</node>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters>--device AudioPlayerWrapper --subdevice portaudioPlayer --name /icub-virtualizer/speakers --playback_network_buffer_lenght 0.4</parameters>
+    <node>icub-virtualizer</node>
+    <dependencies>
+      <port timeout="5.0" request="start" reply="[ok]">/icub/microphone/rpc</port>
+    </dependencies>
+  </module>
+
+  <!-- icub-console-gui -->
+
+  <module>
+    <name>OculusRetargetingModule</name>
+    <node>icub-console-gui</node>
+    <parameters> --OCULUS::move_icub_using_joypad 0 --GENERAL::useSenseGlove 1 </parameters>
+    <dependencies>
+      <port timeout="5.0">/transformServer/transforms:o</port>
+      <port timeout="5.0">/joypadDevice/Oculus/rpc:i</port>
+      <port timeout="5.0">/icub/cam/left</port>
+      <port timeout="5.0">/icub/cam/right</port>
+    </dependencies>
+  </module>
+
+   <module>
+     <name>XsensRetargetingModule</name>
+     <node>icub-console-gui</node>
+   </module>
+
+  <module>
+    <name>HapticGloveModule</name>
+    <node>icub-console-gui</node>
+  </module>
+
+  <module>
+    <name>dyn-visualizer</name>
```

Comment:
Is this module located at iDynTree? About its installation, is it installed by default or needs some options to be enabled?
I found it here https://github.com/S-Dafarra/dyn-visualizer . If it is the one we used, we may add it move it to DIC or Robotology organizations.